packages/evm/contracts/registry/GuardiansRegistry.sol

```diff
@@ -164,6 +179,10 @@ contract GuardiansRegistry is ControlledRecoverable, IGuardiansRegistry, ERC900,
         uint256 minActiveBalance = _getMinActiveBalance(termId);
         require(futureActiveBalance == 0 || futureActiveBalance >= minActiveBalance, ERROR_INVALID_DEACTIVATION_AMOUNT);
 
+        // Check future balance is not bellow the total deactivation locked of the guardian
+        uint256 totalDeactivationLock = guardian.deactivationLocks.total;
```

Comment:
I was thinking about the concept, maybe we want to equate this to "minimum active balance" instead?